projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/handlers.py

```diff
@@ -17,19 +18,17 @@ def get(self):
 
     @tornado.web.authenticated
     def post(self):
-        input_data = self.get_json_body()
-        status_code = VdkUI.run_job(input_data["jobPath"], input_data["jobArguments"])
+        job_data = DictObj(self.get_json_body())
```

Comment:
The `DictObj` could be eventually initialised using any json-
maybe have a singleton instance with a stronger connection/mapping (within the class itself) to always hydrate from the exact json file, then have it hydrated optimally once (since those model properties are rather static)?

Reply:
Also, it is rather strange to populate job_data from DictObject type, maybe consider JobData or similar?

Reply:
@duyguHsnHsn Maybe you can do something like what's suggested here
https://stackoverflow.com/questions/6578986/how-to-convert-json-data-into-a-python-object

Not sure if this is good practice, haven't really had to do this myself. I don't like copying code just because someone claims it works. @ivakoleva what do you think about it?

Reply:
Well I personally think the prior (the change) approach was most simplistic, by accessing directly the input_data properties by key. No unnecessary unique DTOs per function/method to pile up. 
I would only suggest to have a constant of each key, for example usage of the constant:
```
input_data[JOB_NAME]
```
..and the constant is mapped to name, and description (both string).
The description could be used to populate any explanation of what job name is, or job path is.